httpchecker.go

```diff
@@ -54,6 +54,23 @@ type HTTPChecker struct {
 	// requests. If not set, DefaultHTTPClient is
 	// used.
 	Client *http.Client `json:"-"`
+
+	// Headers is a slice of Headerthat contains the
+	// name and value for the header.
+	// Default is no additional headers are sent.
```

Comment:
A little ambiguous; maybe rewrite as "Headers contains headers to set on the request that is sent for the check." or something like that. Also, should the specified headers be Set or Added?